rktlet/image/imagestore.go

```diff
@@ -126,7 +129,7 @@ func (s *ImageStore) ListImages(ctx context.Context, req *runtime.ListImagesRequ
 // PullImage pulls an image into the store
 func (s *ImageStore) PullImage(ctx context.Context, req *runtime.PullImageRequest) (*runtime.PullImageResponse, error) {
 	// TODO auth
-	output, err := s.rkt.RunCommand("image", "fetch", "--no-store=true", "--full=true", "docker://"+*req.Image.Image)
+	output, err := s.RunCommand("image", "fetch", "--no-store=true", "--full=true", "docker://"+*req.Image.Image)
```

Comment:
Good question, several reasons, one is because kubelet will check whether the image already exists. So we should just be as dumb as possible to make sure we DO PULL when kubelet tells us to do it.
Another reason is because currently we can't compare the hash of the docker image with remote so if `--no-store=false` than `rkt fetch docker://busybox:latest` will not fetch the image since it's already in the image store.